src/exo/stdlib/scheduling.py

```diff
@@ -348,56 +351,48 @@ def lift_if(proc, cursor, n_lifts=1):
     return proc
 
 
-# Temporary bounds representation: (idx, base, lo, hi)
-def _get_bounds(bound_repr):
-    _, base, lo, hi = bound_repr
-    return f"{base}+{lo}", f"{base}+{hi}"
-
-
-def _get_size(bound_repr):
-    _, base, lo, hi = bound_repr
-    return hi - lo
-
-
-def _get_bounds_range(bound_repr):
-    _, _, lo, hi = bound_repr
-    return range(lo, hi)
-
-
-def fuse_at(
-    proc, producer, consumer, loop, consumer_bounds, producer_bounds, hardcode=1
-):
+def fuse_at(proc, producer, consumer, loop):
     """
     This version of compute_at will only go down one-level of for loops
 
     TODO: bounds
      - currently assumes that bounds is of the form [0, 1, ..., n-1]
```

Comment:
It seems like this TODO can also be removed since fuse_at no longer takes any bounds arguments.

Reply:
No, it still makes the implicit assumption that the region is contiguous because it generates for loops from `lo` to `hi` without regard for whether all the values from `[lo, hi]` are accessed.